docs/getting-started/create-server-app.html

```diff
@@ -103,12 +103,31 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
     <div id="doc-container" class="docs-item-4 docs-window">
       <main>
         <h1 class="heading-1">Create a server with the Kitura macOS</h1>
-        <h2 class="heading-2" id="prereq">Prerequisites</h2>
-        <ul class="plain-list">
-          <li><a onclick="setActiveSidebarElementForParent('installation-mac')" href="../getting-started/installation-mac.html#install-macos-app">Kitura macOS app:</a> Follow our instructions for installing the Kitura macOS app.</li>
-        </ul>
+        <p class="block-text">The Kitura macOS app provides a user interface (UI) cover for the Kitura CLI. This option is designed for those who wish to avoid using the command line to create a simple cloud ready Kitura project.</p>
```

Comment:
Line 105 in this file (which is a heading) doesn't make sense "Create a server with the Kitura macOS"? I think it's missing the word app